lib/validators/headers-json-example.js

```diff
@@ -1,5 +1,5 @@
 const errors = require('../errors');
-const { JsonSchemaLegacy } = require('./json-schema-legacy');
+const { JsonSchemaValidator } = require('./json-schema-next');
```

Comment:
Since Header validation is based on an implicitly generated JSON Schema Draft 4, we cannot use the legacy class anymore. The legacy class is left only to support Draft 3.